package.json

```diff
@@ -259,6 +259,7 @@
     "request-promise": "4.2.2",
     "semver": "5.6.0",
     "uuid": "^8.3.2",
+    "vscode-azureextensionui": "^0.48.0",
```

Comment:
So far we've avoided using this package to keep the account extension small - especially as it relates to activation times. Can you look into the effects of using this package before we start using it for everything?

Reply:
I did some activation time testing on my machine and adding the UI package doesn't seem to make an appreciable difference (units are ms):

| VSIX with UI package | VSIX without UI package |
|------------|---------------|
| 98         | 65            |
| 65         | 62            |
| 63         | 68            |
| 64         | 68            |
| 65         | 66            |

Reply:
Can you check on Windows as well? If that checks out - I'm okay using the UI package. However we need to keep on eye on this, specifically as we use more and more of the package and as we get telemetry on activation times from real users (we have that telemetry in this extension, right?)